waspc/data/Generator/templates/server/src/routes/operations/index.js

```diff
@@ -6,13 +6,13 @@ import auth from '../../core/auth.js'
 {=/ isAuthEnabled =}
 
 {=# operationRoutes =}
-import {= importIdentifier =} from '{= importPath =}'
+{=& importStatement =}
 {=/ operationRoutes =}
 
 const router = express.Router()
 
 {=# operationRoutes =}
-router.post('{= routePath =}', {=# isUsingAuth =} auth, {=/ isUsingAuth =} {= importIdentifier =})
```

Comment:
Style cleanup: The output went from 
`router.post('bla',  auth,  bla)` to `router.post('bla', auth, bla)`